### PR TITLE
03_Constants and Exercise created

### PR DESCRIPTION
**Pull request by PhoGitFul**
Constants and exercise created at same time, pushing both up I think now....

### CODE REVIEW

**Review by NalisRI — APPROVED** [review]

File: exercise2/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+1. An Identifier is used to "name" a variable or constant, to use the parlance of the term itself, it is used to identify a variable or constant within the program. 
+In the Constant exercise I created 4 identifiers (3 Constants, 1 Variable)
+Constants 'hotelName', 'longitude', 'latitude' and a variable 'occupancy'
+
+2. Identifiers should use camelCase, so should always start with a lowercase character.
```
Comment (NalisRI) [inline exercise2/README.md:5]: if you want to put Q as well, with some bold distinction
#### Q. By which type of character should an identifier begin?
A. Identifiers should use camelCase, so should always start with a lowercase character.

File: exercise2/Notes_on_push.md
[the remark this answers is not in the document]
Comment (PhoGitFul) [reply exercise2/Notes_on_push.md:1]: This wasn't required as part of the exercise but as I had warnings from when I was trying to push and I seemed to get them resolved I took a snapshot of the Terminal commands and responses for my understanding.  I then pushed this up too, and I wondered why it did not ask for a PR, but I notice as this one was still open it just added this file to the open PR.

File: exercise2/README.md (hunk 1 above)
Comment (PhoGitFul) [reply exercise2/README.md]: Hi Srini, yeah good idea.
I had to google how to make the text **bold** so hope it worked.
Amended the file and pushed up anyway.
Shitul could perhaps verify I added the comments correctly :-)

File: exercise2/exeConstant.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"fmt"
+)
+
+func main() {
+	const hotelName string = "Gopher Hotel" //typed constant
+	const longitude = 24.806078             //untyped constant default type to float
+	const latitude = -78.243027             //untyped constant default type to float
+	var occupancy int = 12                  //typed variable
+
+	fmt.Println(hotelName, longitude, latitude, occupancy)
```
Comment (ShitulPatel12) [inline exercise2/exeConstant.go:13]: You have got everything right except for the requirements - which is **"Print the hotelName the longitude, the latitude, and the occupancy on a new line"**

File: exercise2/exeConstant.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	//fmt.Println(reflect.TypeOf(longitude))
+	//fmt.Println(reflect.TypeOf(latitude))
+	//	fmt.Println("Constant Exercise")
+	//	fmt.Println("Hotel:", hotelName, "is located at longitude -", longitude, ",", "latitude -", latitude, ",", "with occupancy of ", occupancy)
+
```
Comment (ShitulPatel12) [inline exercise2/exeConstant.go:18]: I would clear all the commented code if it has done its job

File: exercise2/exeConstant.go (hunk 2 above)
Comment (PhoGitFul) [reply exercise2/exeConstant.go:13]: Oh, I must have misunderstood, did you wish for each field to be printed on its own new line? e.g.
hotelName
longitude
latitude
occupancy
I printed all on one line.

File: exercise2/exeConstant.go (hunk 3 above)
Comment (PhoGitFul) [reply exercise2/exeConstant.go:18]: Yeah I left in so you (and I) could see my "workings out" :-)

File: exercise2/exeConstant.go (hunk 2 above)
Comment (PhoGitFul) [reply exercise2/exeConstant.go]: I made the change and pushed up for review

File: exercise2/exeConstant.go (hunk 3 above)
Comment (PhoGitFul) [reply exercise2/exeConstant.go]: I removed commented out lines also.

File: exercise2/Notes_on_push.md [hunk 4]
```diff
@@ -0,0 +1,377 @@
+mal@ip-10-192-241-136 goLearning % git pull
+Already up to date.
+mal@ip-10-192-241-136 goLearning % git checkout -b 03_Constant
+fatal: a branch named '03_Constant' already exists
+mal@ip-10-192-241-136 goLearning % git branch --set-upstream-to=origin/main 03_Constant
+branch '03_Constant' set up to track 'origin/main'.
+mal@ip-10-192-241-136 goLearning % cd 03_Constants
+mal@ip-10-192-241-136 03_Constants % go run main.go
```
Comment (ShitulPatel12) [inline exercise2/Notes_on_push.md:8]: you can add this file in the .gitignore so it is not commit to your master or repository

Comment (PhoGitFul) [reply exercise2/Notes_on_push.md:8]: Good call, I will look at the .gitignore now and see if I can puzzle it out ;-)

**Pull request merged**